### ISSUE TITLE
Normal Mode Keybindings

### ISSUE THREAD

**Issue by sschepens** [gchp/iota#75]
I was going to implement a few ones but saw that @gchp already did.
I see a few problemas:
- In vim '^' means start of non whitespacetext in line, not start of line, which would skip identations, tabulations, etc.
- Start of line in vim is actually 0 (zero).
- While undo is 'u', redo is actually ctrl+r because 'r' is for replacing characters

**Comment by gchp** [comment]
Yup, you're right about the start of line thing. We don't have a way of finding the first non-whitespace char on a line, however if you want to add those, that would be cool!

I believe I already have Ctrl+r bound to redo?

Modes are still very new, so there's lots to do yet! Thanks for contributing!
